src/ONYXKEYS.ts

```diff
@@ -126,6 +126,9 @@ const ONYXKEYS = {
     /** This NVP contains the choice that the user made on the engagement modal */
     NVP_INTRO_SELECTED: 'introSelected',
 
+    /** The NVP with the last distance rate used per policy */
+    NVP_LAST_SELECTED_DISTANCE_RATES: 'lastSelectedDistanceRates',
```

Comment:
I know it's in the docs this way, but maybe it would be better to make it singular? (NVP_LAST_SELECTED_DISTANCE_RATE) @neil-marcellini wdyt?

Reply:
I think plural makes sense because it's the set of all your last selected distance rates, per policy. When we use it though, we're mostly looking at one policy and one rate, so I understand your question.

Reply:
I think it's fine as is.